QKSMS/src/main/res/values-pl/strings.xml

```diff
@@ -523,4 +515,63 @@
+  <string name="menu_mark_unread">Oznacz jako nieprzeczytane</string>
+  <string name="menu_multi_select">Wycisz rozmowę</string>
+  <string name="menu_notification_settings">Ustawienia powiadomień</string>
+  <string name="menu_view_details">Wyświetl szczegóły</string>
+  <string name="message_delivered">Wiadomość Dostarczona</string>
+  <string name="message_not_delivered">Wiadomość Nie Dostarczona</string>
+  <string name="message_queued">Nie można teraz wysłać wiadomości. Zostanie ona wysłana gdy usługa zostanie wznowiona.</string>
+  <string name="mms_setup_success_title">Udało się!</string>
+  <string name="no_apn">Nie wybrano APN-ów w aplikacji. Popraw to w ustawieniach!</string>
+  <string name="pref_auto_data">Ustawienia danych pakietowych</string>
+  <string name="pref_auto_data_summary">Włącz dane pakietowe automatycznie przy odbieraniu i wysyłaniu MMS</string>
+  <string name="pref_auto_download">Automatyczne pobieranie MMS</string>
+  <string name="pref_auto_download_summary">Automatycznie pobieraj MMS kiedy tylko będą dostępne</string>
+  <string name="pref_auto_emoji_summary">Podmieniaj tekst na emoji</string>
+  <string name="pref_automatically_configure_mms">Ustawienia konfiguracji MMS</string>
+  <string name="pref_background_summary">Dotknij, aby zmienić kolor tła</string>
+  <string name="pref_bubble_style_new">Nowy wygląd</string>
+  <string name="pref_bubbles">Dymki wiadomości</string>
+  <string name="pref_bubbles_summary">Dotknij, aby zmienić ustawienia dymków wiadomości</string>
+  <string name="pref_donate_summary">QKSMS jest bezpłatne. Wspomóż nas przekazując darowiznę!</string>
+  <string name="pref_donate_title">Wspomóż mnie</string>
+  <string name="pref_enter_button">Przycisk enter</string>
+  <string name="pref_enter_button_summary">Dotknij, aby zmienić zachowanie przycisku enter</string>
+  <string name="pref_prox_sensor_call">Czujnik zbliżeniowy</string>
+  <string name="pref_prox_sensor_call_summary">Zbliż telefon do ucha, aby rozpocząć rozmowę jeśli jesteś w trakcie pisania wiadomości</string>
+  <string name="pref_text">Tekst</string>
+  <string name="product_name_donation">Darowizna</string>
+  <string name="service_message_not_found">Wiadomość utraciła ważność lub jest niedostępna.</string>
+  <string name="service_network_problem">Nie udało się wysłać z powodu problemu z siecią.</string>
+  <string name="service_not_activated">Usługa nie jest aktywna w sieci.</string>
+  <string name="stagefright_warning">Wiadomości wideo mogą zawierać szkodliwy kod który jest w stanie zaatakować Twój telefon. Czy chcesz kontynuować?</string>
+  <string name="status_failed">Nie powiodło się. Dotknij, aby zobaczyć opcje</string>
+  <string name="unknown_sender">Nieznany nadawca</string>
+  <string name="update_icon_message">Przy zmianie koloru ikony należy uruchomić ponownie QKSMS. Zrobić to teraz?</string>
+  <string name="want_read_indicators">Want read indicators?</string>
```

Comment:
Would be nice to change this one as well, although I am not sure what's the context